compose.yaml

```diff
@@ -3,42 +3,35 @@ services:
     build: 
       context: .
       target: development
-    ports:
-      - '3002:3002'
-      - '9002:9229'
     links:
       - 'redis:redis'
     depends_on:
       redis:
         condition: service_started
     environment:
-      PORT: 3002
+      PORT: 3000
       NODE_ENV: development
-      ALLOW_ERROR_VIEWS: ${ALLOW_ERROR_VIEWS}
+      ALLOW_ERROR_VIEWS: ${ALLOW_ERROR_VIEWS:-false}
```

Comment:
Adding false to make a clear default for local development. 

Judgement call on what that default should be locally as you can make a valid argument for both.  In the absence of a decision, have opted for the option that takes you closest to production.